src/public/preload.js

```diff
@@ -15,6 +15,29 @@ events.forEach(function (e) {
     });
 });
 
+const userPresenceControl = () => {
+    UserPresence.stopTimer(); //stop userpresence control
+    const AWAY_TIME = 300000; // 5 min
```

Comment:
I wonder if since we are managing we should decrease this.  I assume system idle is going to take into account mouse movements and stuff right?